src/PostgREST/AppState.hs

```diff
@@ -117,6 +124,12 @@ putConfig = atomicWriteIORef . stateConf
 getTime :: AppState -> IO UTCTime
 getTime = stateGetTime
 
+-- | Log with local time
+logWithZTime :: AppState -> Handle -> Text -> IO ()
+logWithZTime appState hdl txt = do
+  zTime <- stateGetZTime appState
+  hPutStrLn hdl $ toS (formatTime defaultTimeLocale "%d/%b/%Y:%T %z: " zTime) <> txt
```

Comment:
Thought AppState was the best place for the log function since it requires its internal state. The time format of the log is similar to the one we use for logging requests(see [here](https://hackage.haskell.org/package/fast-logger-3.0.5/docs/src/System.Log.FastLogger.Date.html#simpleTimeFormat)).